app/views/users/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>user name  <%= @user.name %></h1>
\ No newline at end of file
+<%= render 'shared/user_card', user: @user %>
+<h3>Bio</h3>
+<p><%= @user.bio %></p>
+<% if @user.recent_posts.any? %>
+  <ul>
+    <% @user.recent_posts.each do |post| %>
+      <li>
+      <%= render 'shared/post_card', post: post %>
+      </li>
+    <% end %>
+    <%= link_to "See all posts", user_posts_path(@user) if @user.posts_counter > 0 %>
+    <%= link_to "Add a new post", new_user_post_url(@user) %>
+  </ul>
+<% else %>
```

Comment:
- [ ] In the `users/index` view, it would be beneficial to include a link that directs users to the `user/show` page when they click on any user. This addition would enable users to seamlessly create new posts, comments, and likes for the selected user. Such a feature enhances the user experience and makes it more intuitive for users to interact with the platform.

  - ### YOUR APP BEHAVIOUR

    ![chrome-capture-2023-8-30](https://github.com/souad988/the-x-blog/assets/87197412/58de4491-4c4d-474b-a8ec-dc77f7686647)

Reply:
thanks for your help and for the suggestion @levy002  👍🏼